core/src/main/java/com/amplifyframework/logging/LoggingEventName.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amplifyframework.logging
+
+/**
+ * An enumeration of the names of events relating the {@link LoggingCategory},
```

Comment:
In kdoc these links should just be brackets e.g. `[LoggingCategory]`, see [KDoc syntax](https://kotlinlang.org/docs/kotlin-doc.html#links-to-elements) for details.